test/docker-test/omsfiles/oms_run_script.py

```diff
@@ -93,82 +90,75 @@ def disable_dsc():
+        replace_items(apache_conf_dir, apache_access_conf_path_string, '/var/log/apache2/access.log')
+        replace_items(apache_conf_dir, apache_error_conf_path_string, '/var/log/apache2/error.log')
+        os.system('chmod +r /var/log/apache2/* && chmod +rx /var/log/apache2')
+    elif INSTALLER == 'RPM':
+        replace_items(apache_conf_dir, apache_access_conf_path_string, '/var/log/httpd/access_log')
+        replace_items(apache_conf_dir, apache_error_conf_path_string, '/var/log/httpd/error_log')
+        os.system('chmod +r /var/log/httpd/* && chmod +rx /var/log/httpd')
+
+def inject_logs():
+    if INSTALLER == 'DPKG':
+        os.system('cp /home/temp/omsfiles/apache_access.log /var/log/apache2/access.log')
     elif INSTALLER == 'RPM':
-        replace_items(apache_conf_dir, '<apache-access-dir>', '/var/log/httpd/access_log')
-        replace_items(apache_conf_dir, '<apache-error-dir>', '/var/log/httpd/error_log')
-
-def generate_data():
-    os.system('rm -rf /tmp/apachefake \
-            && git clone https://github.com/kiritbasu/Fake-Apache-Log-Generator /tmp/apachefake \
-            && pip install -r /tmp/apachefake/requirements.txt \
-            && python /tmp/apachefake/apache-fake-log-gen.py -n 100 -o LOG -p /home/temp/omsfiles/web1')
-    if INSTALLER == 'APT':
-        os.system('mv /home/temp/omsfiles/web1_access_log_*.log /var/log/apache2/access.log')
-    elif INSTALLER == 'YUM':
-        os.system('mv /home/temp/omsfiles/web1_access_log_*.log /var/log/httpd/access_log')
-    elif INSTALLER == 'ZYPPER':
-        os.system('mv /home/temp/omsfiles/web1_access_log_*.log /var/log/apache2/access_log')
-
-def start_services():
+        os.system('cp /home/temp/omsfiles/apache_access.log /var/log/httpd/access_log')
+
+    os.system('cp /home/temp/omsfiles/mysql.log /var/log/mysql/mysql.log \
+                && cp /home/temp/omsfiles/error.log /var/log/mysql/error.log \
+                && cp /home/temp/omsfiles/mysql-slow.log /var/log/mysql/mysql-slow.log \
+                && cp /home/temp/omsfiles/custom.log /var/log/custom.log')
+
+def start_system_services():
     os.system('service rsyslog start')
     if INSTALLER == 'DPKG':
         os.system('service cron start \
                     && service apache2 start')
-                    # && service mysql start')
     elif INSTALLER == 'RPM':
         os.system('service crond start \
                     && service httpd start')
-                    # && service mysqld start')
 
 def config_start_oms_services():
     os.system('/opt/omi/bin/omiserver -d')
+    disable_dsc()
     copy_config_files()
     apache_mysql_conf()
-    generate_data()
-    disable_dsc()
-
-def copy_config_files():
-    os.system('dos2unix /home/temp/omsfiles/perf.conf \
-            && dos2unix /home/temp/omsfiles/rsyslog-oms.conf \
-            && dos2unix /home/temp/omsfiles/apache_logs.conf \
-            && dos2unix /home/temp/omsfiles/mysql_logs.conf \
```

Comment:
so now that we deleted mysql_logs.conf and don't copy anything, how do we set config?
i see we copy a mysql_logs.conf from  /etc/opt/microsoft/omsagent/sysconf/omsagent.d/mysql_logs.conf, does that have paths that work by default? (do the default log file paths match where copy the premade log files to on line 136?):

`os.system('cp /home/temp/omsfiles/mysql.log /var/log/mysql/mysql.log \
                && cp /home/temp/omsfiles/error.log /var/log/mysql/error.log \
                && cp /home/temp/omsfiles/mysql-slow.log /var/log/mysql/mysql-slow.log \
                && cp /home/temp/omsfiles/custom.log /var/log/custom.log')`

just want to clarify

Reply:
The default path for the log file is the same as we are setting it up, so there's no need for mysql_logs.conf file any more to be copied from the host. It can just copy the default one present on the container when it get's installed.

Reply:
Perfect, just wasn't immediately sure